library/src/main/java/com/airbnb/android/airmapview/NativeGoogleMapFragment.java

```diff
@@ -138,6 +138,11 @@ public void onInfoWindowClick(Marker marker) {
     callback.onMapBoundsReady(builder.build());
   }
 
+  @Override
+  public void getLatLngScreenLocation(LatLng latLng, OnLatLngScreenLocationCallback callback) {
```

Comment:
the callback class' name can remain the same - helpful to have `LatLng` there since it isn't as obvious without it